actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -101,13 +103,15 @@ def localize(object, **options)
       alias :l :localize
 
       private
+        MISSING_TRANSLATION = Object.new
+        private_constant :MISSING_TRANSLATION
+
         def scope_key_by_partial(key)
-          stringified_key = key.to_s
-          if stringified_key.start_with?(".")
+          if key.start_with?(".")
```

Comment:
It looks like this relies on our backport of `Symbol#start_with?` to work on Ruby < 2.7, so this file should now require `active_support/core_ext/symbol/starts_ends_with`.

Reply:
Good eyes, as always!  :eagle: :eyes: